src/xo-app/user/index.js

```diff
@@ -226,12 +227,43 @@ class UserFilters extends Component {
 }
 
 // ===================================================================
+const COLUMNS = [
+  {
+    default: true,
+    itemRenderer: sshKey => sshKey.title,
+    name: _('title'),
+    sortCriteria: 'title',
+  },
+  {
+    itemRenderer: sshKey => <span style={SSH_KEY_STYLE}>{sshKey.key}</span>,
+    name: _('key'),
+  },
+]
+
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: deleteSshKey,
+    icon: 'delete',
+    label: _('deleteSshKey'),
+  },
+]
+
+const GROUPED_ACTIONS = [
+  {
+    handler: deleteSshKeys,
+    icon: 'delete',
+    label: _('deleteSshKeys'),
+  },
+]
 
 const SshKeys = addSubscriptions({
   user: subscribeCurrentUser,
 })(({ user }) => {
   const sshKeys = user && user.preferences && user.preferences.sshKeys
 
+  // sshKeysWithIds variable contains ssh keys Ids to be passed to SortedTable.
```

Comment:
No, it contains SSH keys **objects** augmented with an `id` property required by `SortedTable`.